app/views/groups/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= render partial: "navbar" %>
+<div id="groups">
+  <% @groups.each do |group| %>
+    <ul class="group--section">
+    <li class= "single__group__item">
+    <%= link_to group_path(group.id), class: "single__group__item__link" do %>
+    <div class="group__container">
+      <div class="group__items">
+        <div class="group__icon">
+          <img class="category-image" src="<%= group.icon %>" alt="Group Icon" width="120" height="150">
+        </div>
+        <div class="group__properties">
+          <div class="group__details">
+            <h4><%= group.name %></h4>
+            <p>$<%= group.payments.sum(:amount)%></p>
+          </div>
+          <div class="group__utilities">
+            <h6>Created at: <span><%= group.created_at.strftime("%d %B, %Y") %></span></h6>
+          </div>
```

Comment:
- [ ] Good job so far 👏 👏 You've done great job! In addition to what you have done, I would suggest that you improve the home UI by adjusting the alignment of the group name and its icon. Please look at the following comparison 👇 

**Your design**
![image](https://user-images.githubusercontent.com/90222110/230640047-34aced00-4d5d-4a45-8034-f432162dd955.png)

**Design Given**
![image](https://user-images.githubusercontent.com/90222110/230640171-d93b06d0-9893-49e2-a5e0-effaabab5b41.png)

As you can see,  the group name and icon are supposed to be  aligned in a row not in the column so please update it

- [ ] From the comparison above, you can see that the amount on the design given is in bold, therefore I would suggest that you make it bold in your design too but also try to increase it's font-size in proportion to the size of the card in order for it to be readable 😃 😎